checkstyle.xml

```diff
@@ -36,11 +36,9 @@ limitations under the License.
     <module name="RedundantImport"/>
     <module name="UnusedImports"/>
     <module name="NeedBraces"/>
-    <module name="RedundantThrows">
```

Comment:
Yes. I explained this in the relevant commit (a7d7e37d091bf1ba3674c6fc617466b284aaac71) , but perhaps I should add the explanation to the PR's comment too, to make it more obvious. New checkstyle versions removed this check, so we can no longer use it. Since checkstyle is not type-aware, there's really no way of performing this check properly - see https://github.com/checkstyle/checkstyle/issues/473 for the discussion about this removal.